recipes/netpbm/all/conanfile.py

```diff
@@ -0,0 +1,238 @@
+
+        def _use_pkgconfig(name):
+            config.append(f"{name}LIB=USE_PKGCONFIG.a")
+            config.append(f"{name}HDR_DIR=USE_PKGCONFIG.a")
+
+        if self.options.tools:
+            _configure_dependency("TIFF", "libtiff", "tiff")
+            _configure_dependency("PNG", "libpng", "png")
+            _configure_dependency("JPEG", str(self.options.with_libjpeg), "jpeg")
+            _configure_dependency("JASPER", "jasper", "jasper")
+            _configure_dependency("JBIG", "jbig", "jbig")
+            if self.options.get_safe("with_x11"):
+                _use_pkgconfig("X11")
+
+        cflags = list(tc.cflags)
+        if self.settings.build_type == "Debug":
+            cflags += ["-g"]
+        else:
+            # Match flags set by ./configure
+            cflags += ["-O3", "-ffast-math", "-fno-finite-math-only"]
+        if self.options.get_safe("fPIC", True):
+            cflags.append("-fPIC")
+        if is_apple_os(self):
+            # For vasprintf
+            cflags.append("-D_DARWIN_C_SOURCE")
+        config.append(f"CFLAGS={' '.join(cflags)}")
+
+        tc.make_args.extend(config)
+        tc.generate()
+
+        deps = PkgConfigDeps(self)
+        deps.generate()
+
+    def _patch_sources(self):
+        # Fix FILE not being defined in jpeglib.h
+        replace_in_file(self, os.path.join(self.source_folder, "converter", "other", "cameratopam", "camera.c"),
+                        "#include <jpeglib.h>", "#include <stdio.h>\n#include <jpeglib.h>")
+        # add missing -ljpeg to cameratopam
+        save(self, os.path.join(self.source_folder, "converter", "other", "cameratopam", "Makefile"),
+             "\ncameratopam: LDFLAGS_TARGET = $(shell $(LIBOPT) $(LIBOPTR) $(JPEGLIB))\n", append=True)
+
+        if not self.options.tools:
+            replace_in_file(self, os.path.join(self.source_folder, "GNUmakefile"),
+                            "PROG_SUBDIRS =", "PROG_SUBDIRS = #")
+
+    def build(self):
+        self._patch_sources()
+        with chdir(self, self.source_folder):
+            shutil.copy("config.mk.in", "config.mk")
+            autotools = Autotools(self)
+            autotools.make()
+
+    def package(self):
+        for file in ["copyright_summary", "patent_summary", "GPL_LICENSE.txt", "lgpl_v21.txt", "COPYRIGHT.PATENT"]:
+            copy(self, file, os.path.join(self.source_folder, "doc"), os.path.join(self.package_folder, "licenses"))
+        temp_dir = self.package_path.joinpath("tmp")
+        lib_dir = temp_dir.joinpath("lib")
+        with chdir(self, self.source_folder):
+            autotools = Autotools(self)
+            autotools.make(target="package", args=[f"pkgdir={temp_dir}"])
+        if self.options.shared:
+            if is_apple_os(self):
+                soname_file = next(lib_dir.glob("libnetpbm.*.*.dylib"))
+                rename(self, soname_file, lib_dir.joinpath("libnetpbm.dylib"))
+                rm(self, "libnetpbm.*.dylib", lib_dir)
```

Comment:
Any insight you could give int this? thanks :)

Reply:
The build scripts in the project incorrectly configure the output name to include soname numbers even on macOS. This renames it and removes an additional symlink file.